src/game/lara.c

```diff
@@ -2264,9 +2389,11 @@ int32_t LaraLandedBad(ITEM_INFO *item, COLL_INFO *coll)
     item->floor = height;
     item->pos.y = height;
     TestTriggers(TriggerIndex, 0);
-    item->pos.y = oy;
+    item->pos.y = oy; // restores Y so don't touch double y
```

Comment:
I don't make a separate cache for the double pos.Y because this restores Y and the code in-between doesn't modify the floating point position. I.e this only does fp Y not f Y because fp Y is ultimately unchanged.